hello-typescript/src/task3.ts

```diff
@@ -0,0 +1,34 @@
+export default function sortObjectByDate<T extends {createdAt:string,updatedAt?:string}>(array: T[]):T[] {
+
+    let result = array.slice();
```

Comment:
از این Syntax استفاده کنید که کاربردش کپی‌گرفتنه:

```suggestion
    let result = [...array]
```